### PR TITLE
Adding scoped reference to 'can'. Removing dependancy on Global 'can'.

### PR DESCRIPTION
**Pull request by Psykoral**

### CODE REVIEW

File: list/promise/promise.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-steal("can/list", function (list) {
```
Comment (Psykoral) [inline list/promise/promise.js]: In this case, 'list' was referenced but never used. 'can' was not referenced but used.

**Comment by justinbmeyer** [comment]
Thanks!

**Comment by Psykoral** [comment]
Fixing that last build error, give me a few

**Comment by justinbmeyer** [comment]
I'm not sure what's breaking: https://travis-ci.org/bitovi/canjs/jobs/87143498#L2388

Anyway you an check it out?

The easy way is to run:

```
grunt testify
```

And then open `test/amd/jquery-2.html` and see if that's breaking in the browser.

**Comment by Psykoral** [comment]
It looks like can.isPlainObject is defined in https://github.com/bitovi/canjs/blob/master/util/object/isplain/isplain.js#L29 and so the new local scoped can doesn't know what it is.

**Comment by Psykoral** [comment]
:+1:  :100:

**Comment by daffl** [comment]
Awesome, thank you!

**Comment by Psykoral** [comment]
My pleasure!

**Pull request merged**